crates/net/downloaders/src/bodies/bodies.rs

```diff
@@ -237,13 +266,24 @@ where
             }
 
             // Drop buffered response since we passed that range
-            if *next_block_rng.end() < expected {
-                self.metrics.buffered_responses.decrement(1.);
-                self.buffered_responses.pop();
+            if *next_block_range.end() < expected {
+                self.pop_buffered_response();
             }
         }
         None
     }
+
+    /// Returns the next batch of block bodies that can be returned if we have enough buffered
+    /// bodies
+    fn try_split_next_batch(&mut self) -> Option<Vec<BlockResponse>> {
+        if self.queued_bodies.len() >= self.stream_batch_size {
+            let next_batch = self.queued_bodies.drain(..self.stream_batch_size).collect::<Vec<_>>();
+            self.metrics.total_flushed.increment(next_batch.len() as u64);
+            self.metrics.queued_blocks.set(self.queued_bodies.len() as f64);
```

Comment:
This queued block metric is the same as the flushed metric, we should remove either one and update the Grafana dashboard

Reply:
flushed only increments but `queued_blocks` is always pinned to the vec's length, so they are different